build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+	id("org.springframework.boot") version "2.3.4.RELEASE"
+	id("io.spring.dependency-management") version "1.0.10.RELEASE"
+	kotlin("jvm") version "1.4.10"
+	kotlin("plugin.spring") version "1.4.10"
+}
+
+group = "com.kostas"
+version = ""
+java.sourceCompatibility = JavaVersion.VERSION_12
+
+repositories {
+	mavenCentral()
+	maven {
+		url = uri("http://packages.confluent.io/maven/")
+	}
+}
+
+object Versions {
+	const val kafkaClients = "2.6.0"
+	const val springKafka = "2.6.2"
+	const val avro = "1.10.0"
+	const val avroSerializer = "6.0.0"
```

Comment:
tut tut have you copied CMS? 

Dont need most of this

Reply:
I added these here on the early days I wanted to use Avro before I decide not to.. But yeah, they are not in use now.